target/linux/generic/hack-6.12/931-mips-ralink-add-missing-include.patch

```diff
@@ -0,0 +1,17 @@
+From b71ab42067204c3d90aa9143b815482b222fee32 Mon Sep 17 00:00:00 2001
```

Comment:
Why is this a "hack"? If this issue is introduced by one of our patches, the fix should go in there, if it is upstream, it should be sent upstream. And either way it isn't a hack.

Reply:
If not put the fix in the hack then where?

Reply:
So please send this patch to upstream.

Reply:
As @KanjiMonster said, You are putting this in a wrong place, it should be added in `target/linux/ramips/patches-*/311-MIPS-use-set_mode-to-enable-disable-the-cevt-r4k-irq.patch`.

Reply:
Ok I see.